### ISSUE TITLE
RadTool | Identify duplicate column

### ISSUE THREAD

**Issue by ashgadala** [aspnetzero/documents#94]
RAD Tool should be able to identify any of the duplicate column while created or regeneration of the entity.
As of the latest version it does not.

**Comment by yekalkan** [comment]
It already checks the column names. It doesn't check only sameness of a primitive property and a navigation property.

**Comment by ismcagdas** [comment]
@ashgadala could you share a scenario which we can reproduce it ?

**Comment by ashgadala** [comment]
I was creating an entity with about 10 columns.

I did it correctly for the first time. I was testing and wanted to see the default layout for the existing datatypes. 

So I regenerated the entity from pick-list and updated namespaces,entity name  etc to new one, 
Then updated the column names to sentence case and updated bool datatype to byte.While doing so i renamed the two of column names to same. Clicked generate.Rad tool generated all UI,Dto's view-models and everything that it usually do but the migrations failed as expected because of duplicate column. 

I hope this make sense. If not let me know Happy to skype or hangout etc.
